ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/external-lb/provision/cucushift-installer-rehearse-nutanix-ipi-external-lb-provision-chain.yaml

```diff
@@ -0,0 +1,15 @@
+chain:
+  as: cucushift-installer-rehearse-nutanix-ipi-external-lb-provision
+  steps:
+  - chain: ipi-conf-nutanix
+  - ref: ignition-bastionhost
+  - ref: ignition-bastionhost-haproxy
+  - ref: nutanix-provision-bastionhost
+  - ref: ipi-conf-nutanix-zones
```

Comment:
if the external lb is working with multi zones, the install workflow should be placed in `ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/zones/` as a subfolder.

Reply:
I confirmed with @zzhao, `cucushift-installer-rehearse-nutanix-ipi-external-lb` works on both zone and non-zone env, we selected zone env to run it

Reply:
But for this workflow, `cucushift-installer-rehearse-nutanix-ipi-external-lb` is using zone env, right?

Reply:
Got it, I'll move this workflow